app/controllers/api_entreprise/restricted_token_magic_links_controller.rb

```diff
@@ -22,6 +17,14 @@ def create
 
   private
 
+  def organizer
+    Token::DeliverMagicLinkToEmail.call(
```

Comment:
```suggestion
    @organizer ||= Token::DeliverMagicLinkToEmail.call(
```

cela évite de le call potentiellement N fois

en vrai je préfère l'ancienne version qui est plus rails way

Reply:
l'ancienne version me faisait une erreur rubocop à cause de la complexité donc je préfère laisser dans la méthode à part